README.md

```diff
@@ -50,6 +50,15 @@ See these pages for more info:
 
 [Github API OAuth Overview] (http://developer.github.com/v3/oauth)
 
+Enterprise Github instances may be specified using the `apiUrl` option:
```

Comment:
Hey @aendrew - I was wondering where this part went inside the README on master?